firmware/application/apps/pocsag_app.hpp

```diff
@@ -65,7 +65,14 @@ class POCSAGAppView : public View {
     NavigationView& nav_;
     RxRadioState radio_state_{};
     app_settings::SettingsManager settings_{
-        "rx_pocsag", app_settings::Mode::RX};
+        "rx_pocsag",
+        app_settings::Mode::RX};
+
+    // Settings
+    bool enable_logging = false;
```

Comment:
Another nice benefit of this approach is that the default value can just be defined inline at the declaration.